README.md

```diff
@@ -53,10 +54,10 @@ export default Ember.Route.extend(CanMixin, {
 
 ## Installation
 
-Install this addon via npm:
+Install this addon via ember-cli:
 
 ```
-npm install --save-dev ember-can
+ember install ember-can
```

Comment:
It's important to `ember install` now, because it runs the addon's blueprint, making the change to the resolver that allows us to test abilities.